pkg/kubelet/cadvisor/helpers_linux.go

```diff
@@ -42,7 +42,7 @@ func (i *imageFsInfoProvider) ImageFsInfoLabel() (string, error) {
 		// This is a temporary workaround to get stats for cri-o from cadvisor
 		// and should be removed.
 		// Related to https://github.com/kubernetes/kubernetes/issues/51798
-		if i.runtimeEndpoint == CrioSocket {
+		if i.runtimeEndpoint == CrioSocket || i.runtimeEndpoint == "unix://"+CrioSocket {
```

Comment:
As per https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/ , the runtimeEndpoint can be both tcp and unix.  So with that in mind , is pre-pending `unix://` still the right thing to do ?

```
--container-runtime-endpoint string
-- | [Experimental] The endpoint of remote runtime service. Currently unix socket is supported on Linux, and tcp is supported on windows.
```

Reply:
@krmayankk Thank you for your comment.
As you mentioned, `runtimeEndpoint` supports both unix(on linux) and tcp(on windows).
I think that this "helpers_linux.go" is used by only linux platform.

Reply:
good point @makocchi-git